help/en/html/tools/scripting/HowTo.shtml

```diff
@@ -121,7 +121,7 @@
 		
 	  <div class="para">
 		
-		<p>All of the JMRI input and output objects listed in <a href="help/en/html/tools/index.shtml">
+		<p>All of the JMRI input and output objects listed in <a href="http://jmri.org/help/en/html/tools/index.shtml">
```

Comment:
You need to remove `http://jmri.org/` from here. Otherwise the local copy of this link included in JMRI will not be used in the JMRI help browser.